1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,30 @@
 // Write your code here
+const arrOfCats = ["cute-cat-a.png", "cute-cat-b.png", "cute-cat-c.png"]; 
+const curentImage = document.querySelector("#curentImage");
+const backBtn = document.querySelector("#backBtn");
+const forwardBtn = document.querySelector("#forwardBtn");
+
+let currentIndex = 0;
+
+function backFunction () {
+  currentIndex--
```

Comment:
What if we click back button and  the currentIndex  is 0 already? it will give a negative number and `arrOfCats[-1]` would give us trouble.
Could you think of a way to bring the currentIndex to point to the last cat pic?